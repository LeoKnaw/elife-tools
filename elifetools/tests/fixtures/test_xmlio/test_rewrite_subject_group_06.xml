<article><front><article-meta><article-categories><subj-group subj-group-type="display-channel"><subject>Research Article</subject></subj-group></article-categories></article-meta></front></article>