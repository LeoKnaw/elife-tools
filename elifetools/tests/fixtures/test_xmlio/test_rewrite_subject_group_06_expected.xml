<article><front><article-meta><article-categories /></article-meta></front></article>